components/KickoffRequestView/KickoffPostUrl.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button, TextInput, Grid } from '@mantine/core';
+
+export default function KickoffPostUrl() {
+  return (
+    <Grid>
+      <Grid.Col span={2}>
+        <Button fullWidth>POST</Button>
+      </Grid.Col>
+      <Grid.Col span={10}>
+        <TextInput
+          disabled
```

Comment:
another option here is to use the readOnly prop, as is done in the filters tab, which won't grey out the input, but I'll leave that decision to you

Reply:
Not sure how I missed the `readOnly` prop. It looks much better. Changed to that in bb84cdb.